README.md

```diff
@@ -123,62 +95,46 @@ gcloud auth application-default login
 
 ### Configure gcloud settings
 
+The scripts that manage the infrastructure and deployments for this demo attempt to initialize themselves with values from the `gcloud` config defaults for `compute/zone`, and `core/project`.
+
 Run `gcloud config list` and make sure that `compute/zone`, `compute/region` and `core/project` are populated with values that work for you. You can set their values with the following commands:
 
-Where the region is us-west1:
+Where the region is `us-west1`:
 ```console
 
 gcloud config set compute/region us-west1
 
 Updated property [compute/region].
 ```
 
- Where the zone inside the region is us-west1-a:
+ Where the zone inside the region is `us-west1-a`:
 ```console
 
 gcloud config set compute/zone us-west1-a
 
 Updated property [compute/zone].
 ```
- here the project name is my-project-name:
+ Where the project name is `my-project-name`:
 ```console
 
 gcloud config set project my-project-name
 
 Updated property [core/project].
 ```
 
-### Setup this project
-
-This project can be set up simply by executing
-```console
-./setup.sh
-```
-from the root of the project.
-It will prompt once to enable the required APIs in GCP,
-enter `y` when prompted.
-
-### Configuration
-
-The scripts will attempt to initialize themselves with values from the gcloud
-config defaults for `compute/zone`, and `core/project`.  If they are not already
-set, you can set them with the following commands:
+## Deployment
 
+The infrastructure required by this project can be deployed by executing:
 ```console
-gcloud config set compute/zone <your-zone>
-gcloud config set core/project <your-project>
+make create
 ```
 
-## Deployment
-
-At the root of this repository, execute `setup.sh`. It will:
-1.  Package the deployable Prime-flask application.
-1.  Create the container image and push it to the private image repository
-    in your project.
-1.  Generate variable defaults for the Terraform execution.
+This will:
+1.  Package the deployable `Prime-flask` application.
+1.  Create the container image and push it to the private [Container Registry (GCR)](https://cloud.google.com/container-registry/) for your project.
+1.  Generate variable defaults for the [Terraform](terraform.io) execution.
```

Comment:
generate appropriate configuration for terraform.

Reply:
k